src/XeroPHP/Models/Accounting/Attachment.php

```diff
@@ -201,4 +201,11 @@ public function setLocalHandle($local_handle) {
         $this->local_handle = $local_handle;
     }
 
+    /**
+     * @return mixed
+     */
+    static function isPageable() {
```

Comment:
Good spot. Rather lazily, I copied the function from the abstract class and didn't check. Will resubmit a new PR.